changelogs/current.yaml

```diff
@@ -62,6 +62,8 @@ new_features:
 - area: http
   change: |
     added default-false ``envoy.reloadable_features.http1_use_balsa_parser`` for experimental BalsaParser.
+  change: |
+    added ``envoy.reloadable_features.allow_upstream_filters`` for experimental upstream filters.
```

Comment:
Can we get some stuff added to arch overview somewhere? Fine if you want to do this as a follow up if people are eager to land this. I'm thinking we should talk about this in the general HTTP section and maybe have a configuration overview sample for some upstream use case? With all the existing filters I'm sure we can come up with something?